sites/main-site/src/content/blog/2024/seqera-containers-part-1.mdx

```diff
@@ -0,0 +1,306 @@
+    <tbody>
+        <tr>
+            <td>Support Bioconda packages</td>
+            <td>✅</td>
+            <td>✅</td>
+            <td>✅</td>
+        </tr>
+        <tr>
+            <td>Support all conda channels</td>
+            <td>❌</td>
+            <td>✅</td>
+            <td>✅</td>
+        </tr>
+        <tr>
+            <td>Support PyPI (pip) packages</td>
+            <td>❌</td>
+            <td>✅</td>
+            <td>✅</td>
+        </tr>
+        <tr>
+            <td>Docker + Singularity support</td>
+            <td>✅</td>
+            <td>✅</td>
+            <td>✅</td>
+        </tr>
+        <tr>
+            <td>Linux aarch64 and arm64</td>
+            <td>⏳ In progress</td>
+            <td>✅</td>
+            <td>✅</td>
+        </tr>
+        <tr>
+            <td>Multi-package containers</td>
+            <td>✅ Mulled</td>
+            <td>✅</td>
+            <td>✅</td>
+        </tr>
+        <tr>
+            <td>Conda lock files generated</td>
+            <td>❌</td>
+            <td>✅</td>
+            <td>✅</td>
+        </tr>
+        <tr>
+            <td>Container build logs</td>
+            <td>❌ CI logs short lived</td>
+            <td>✅</td>
+            <td>✅</td>
+        </tr>
+        <tr>
+            <td>Docker container security scans</td>
+            <td>✅ quay.io</td>
+            <td>✅ Trivy</td>
+            <td>✅ Trivy</td>
+        </tr>
+        <tr>
+            <td>SBOM manifests (software bill of materials)</td>
+            <td>❌</td>
+            <td>✅</td>
+            <td>✅</td>
+        </tr>
+        <tr>
+            <td>Long storage duration</td>
+            <td>✅ *</td>
+            <td>❌ 72 hours cache</td>
+            <td>✅ *</td>
+        </tr>
+        <tr>
+            <td>Pull delay for conda packages</td>
+            <td>✅ instant</td>
+            <td>❌ ~2-3 minutes for build on first request</td>
+            <td>✅ instant</td>
+        </tr>
+        <tr>
+            <td>Stable image URIs</td>
+            <td>✅</td>
+            <td>❌ Single-run</td>
+            <td>✅</td>
+        </tr>
+        <tr>
+            <td>Software required for end user</td>
+            <td>Docker / Singularity</td>
+            <td>Wave CLI / Nextflow, Docker / Singularity</td>
+            <td>Docker / Singularity</td>
+        </tr>
+        <tr>
+            <td>Offline support with downloaded images</td>
+            <td>✅</td>
+            <td>❌</td>
+            <td>✅</td>
+        </tr>
+        <tr>
+            <td>Guaranteed identical conda builds in future image pulls</td>
+            <td>✅</td>
+            <td>❌</td>
+            <td>✅</td>
+        </tr>
+    </tbody>
+</table>
+
+> \* Forever is a long time, and we're not fortune tellers. To help the community trust the program, Seqera is committing to keeping images for a minimum of 5 years after they're built. BioContainers has no public policy, images will presumably remain until quay.io / Galaxy remove support.
```

Comment:
does this commitment of keep for min 5yrs start from the creation of a container ?

Reply:
Yup, see FAQ on https://seqera.io/containers/ :

![CleanShot 2024-09-12 at 00 04 14@2x](https://github.com/user-attachments/assets/a4d4c91b-fa19-4f1d-a9a1-70f6665f298c)

Would "creation" be clearer that "built"?

```suggestion
> \* Forever is a long time, and we're not fortune tellers. To help the community trust the program, Seqera is committing to keeping images for a minimum of 5 years from the time of their creation. BioContainers has no public policy, images will presumably remain until quay.io / Galaxy remove support.
```